client/src/pages/Report/NotificationSection/NotificationSection.tsx

```diff
@@ -55,8 +37,24 @@ const NotificationSection = () => {
         setShowModal(false);
     };
 
-    const handleDelete = (id: string) => {
-        setRows((prevRows) => prevRows.filter((row) => row.id !== id));
+    const handleDelete = (id: string, service: string) => {
+        setRows((prevRows) => {
+            // eslint-disable-next-line max-len
```

Comment:
Instead of using eslint-disable-next-line try to make the line shorter, perhaps splitting it into multiple lines